pyproject.toml

```diff
@@ -9,4 +9,4 @@ author-email = "nils.braun@kit.edu"
 home-page = "https://github.com/nils-braun/b2luigi"
 classifiers = ["License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)"]
 
-requires=['luigi<2.7.9', 'parse>=1.8.4', 'GitPython>=2.1.11', "colorama>=0.3.9", "cachetools>=2.1.0"]
+requires=['luigi<2.7.9', 'parse>=1.8.4', 'GitPython>=2.1.11', "colorama>=0.3.9", "cachetools>=2.1.0", "jinja2"]
```

Comment:
Maybe I should also add a version number to my jinja2 requirement? I use it in one place and its a really basic use-case, I would expect it to work with all versions, but I don't know.